docs/maintainer/infrastructure.md

```diff
@@ -388,3 +471,186 @@ If this is not done, then the output validation process will block the package f
+
+- [`conda-forge`](https://github.com/conda-forge): the main organization
+- [`regro`](https://github.com/regro): hosts the `autotick-bot` machinery
+- [`channel-mirrors`](https://github.com/channel-mirrors): OCI mirror of the conda-forge channel
+
+:::info
+These organizations exist but they are not in active use anymore:
+
+- [`conda-forge-abandoned`](https://github.com/conda-forge-abandoned)
+- [`conda-forge-woodpecker-admins`](https://github.com/conda-forge-woodpecker-admins)
+
+:::
+
+#### Teams
+
+The `conda-forge` Github organization has thousands of teams.
+Most of them are associated with a feedstock, but there are a few special ones that are not!
+
+<!-- TODO: The following list is prone to getting out of date. We should generate it automatically. -->
+
+- [`Core`](https://github.com/orgs/conda-forge/teams/Core)
+- [`staged-recipes`](https://github.com/orgs/conda-forge/teams/staged-recipes)
+- [`help-c-cpp`](https://github.com/orgs/conda-forge/teams/help-c-cpp)
+- [`help-cdts`](https://github.com/orgs/conda-forge/teams/help-cdts)
+- [`help-go`](https://github.com/orgs/conda-forge/teams/help-go)
+- [`help-java`](https://github.com/orgs/conda-forge/teams/help-java)
+- [`help-julia`](https://github.com/orgs/conda-forge/teams/help-julia)
+- [`help-nodejs`](https://github.com/orgs/conda-forge/teams/help-nodejs)
+- [`help-osx-arm64`](https://github.com/orgs/conda-forge/teams/help-osx-arm64)
+- [`help-perl`](https://github.com/orgs/conda-forge/teams/help-perl)
+- [`help-ppc64le`](https://github.com/orgs/conda-forge/teams/help-ppc64le)
+- [`help-pypy`](https://github.com/orgs/conda-forge/teams/help-pypy)
+- [`help-python`](https://github.com/orgs/conda-forge/teams/help-python)
+- [`help-python-c`](https://github.com/orgs/conda-forge/teams/help-python-c)
+- [`help-r`](https://github.com/orgs/conda-forge/teams/help-r)
+- [`help-ruby`](https://github.com/orgs/conda-forge/teams/help-ruby)
+- [`help-rust`](https://github.com/orgs/conda-forge/teams/help-rust)
+- [`miniforge`](https://github.com/orgs/conda-forge/teams/miniforge)
+- [`all-members`](https://github.com/orgs/conda-forge/teams/all-members)
+
+#### Configuration
+
+- [`conda-forge/.github`](https://github.com/conda-forge/.github): Organization-wide configuration, profile information, etc.
+- [`conda-forge/.cirun`](https://github.com/conda-forge/.cirun): Organization-wide configuration for special ci runners (e.g. GPU), c.f. [Cirun](#cirun).
+
+#### Bot accounts
+
+- [`conda-forge-admin`](https://github.com/conda-forge-admin)
+- [`conda-forge-bot`](https://github.com/conda-forge-bot)
+- [`conda-forge-coordinator`](https://github.com/conda-forge-coordinator)
+- [`conda-forge-daemon`](https://github.com/conda-forge-daemon)
+- [`conda-forge-linter`](https://github.com/conda-forge-linter)
+- [`conda-forge-manager`](https://github.com/conda-forge-manager)
+- [`conda-forge-status`](https://github.com/conda-forge-status)
+- [`regro-cf-autotick-bot`](https://github.com/regro-cf-autotick-bot)
```

Comment:
-bot, -coordinator, -manager, and -status are all no longer used

I need to check on -linter